alignak/objects/contact.py

```diff
@@ -202,11 +238,13 @@ def want_service_notification(self, notifways, timeperiods, downtimes,
         if not self.service_notifications_enabled:
             return False
 
-        # If we are in downtime, we do nto want notification
+        # If we are in downtime, we do not want notification
         for downtime_id in self.downtimes:
             downtime = downtimes[downtime_id]
             if downtime.is_in_effect:
+                self.in_scheduled_downtime = True
```

Comment:
Not sure it's the best place, it's better to define it when add downtime / delete a downtime on an item (create a function if not exist)